docs/src/modules/components/MarkdownDocsFooter.js

```diff
@@ -0,0 +1,323 @@
+/* eslint-disable no-restricted-globals */
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { exactProp } from '@material-ui/utils';
+import { withStyles } from '@material-ui/core/styles';
+import DialogActions from '@material-ui/core/DialogActions';
+import TextField from '@material-ui/core/TextField';
+import Collapse from '@material-ui/core/Collapse';
+import Button from '@material-ui/core/Button';
+import Divider from '@material-ui/core/Divider';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+import IconButton from '@material-ui/core/IconButton';
+import ThumbUpIcon from '@material-ui/icons/ThumbUpAlt';
+import ThumbDownIcon from '@material-ui/icons/ThumbDownAlt';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import Snackbar from '@material-ui/core/Snackbar';
+import { getCookie, pageToTitleI18n } from 'docs/src/modules/utils/helpers';
+import PageContext from 'docs/src/modules/components/PageContext';
+import Link from 'docs/src/modules/components/Link';
+
+const styles = (theme) => ({
+  root: {
+    marginTop: theme.spacing(12),
+  },
+  pagination: {
+    margin: theme.spacing(3, 0, 4),
+    display: 'flex',
+    justifyContent: 'space-between',
+    [theme.breakpoints.down('sm')]: {
+      flexWrap: 'wrap',
+    },
+  },
+  pageLinkButton: {
+    textTransform: 'none',
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+  feedbackMessage: {
+    margin: theme.spacing(0, 2),
+  },
+  feedback: {
+    width: 'auto',
+    [theme.breakpoints.down('sm')]: {
+      order: 3,
+      width: '100%',
+    },
+  },
+  hidden: {
+    ariaHidden: 'true',
+    opacity: 0,
+  },
+});
+
+function flattenPages(pages, current = []) {
+  return pages.reduce((items, item) => {
+    if (item.children && item.children.length > 1) {
+      items = flattenPages(item.children, items);
+    } else {
+      items.push(item.children && item.children.length === 1 ? item.children[0] : item);
+    }
+    return items;
+  }, current);
+}
+
+// To replace with .findIndex() once we stop IE11 support.
+function findIndex(array, comp) {
+  for (let i = 0; i < array.length; i += 1) {
+    if (comp(array[i])) {
+      return i;
+    }
+  }
+
+  return -1;
+}
+
+async function postFeedback(data) {
+  const env = location.hostname === 'material-ui.com' ? 'prod' : 'dev';
```

Comment:
If we want to use the exact same test as the service worker:
```suggestion
  const env = window.location.host.indexOf('material-ui.com') <= 0 ? 'prod' : 'dev';
```

Reply:
I was targeting material-ui.com exclusively (next would go to dev), but I guess now that we include the version number we can capture early feedback on next.

However I don't understand the operator used. Surely `<= 0` means that the test string f('material-ui.com') matches at the beginning, or doesn't match at all, and in both cases this goes to prod?

Reply:
Yeah, I'm not sure either about the validity of the condition. How about we use the one your have proposed in the two cases?

Reply:
Do we need to enable it in next? I would say yes, to collect early feedback?

Reply:
What about statically branching depending on `process.env.PULL_REQUEST`? We're already using that for other optimizations. Which also feeds into my earlier suggestions to include the netlify deploy id.

Otherwise we could get some bias on faulty deploys.

Reply:
Any feedback on a pull request deploy preview is going to go to the dev table anyway, where it can safely be ignored.

Reply:
> Any feedback on a pull request deploy preview is going to go to the dev table anyway, where it can safely be ignored.

So it sounds like we can consolidate the split at one location.

Reply:
Sorry, I don't understand